client/src/app/home/home.component.ts

```diff
@@ -30,9 +30,11 @@ export class HomeComponent implements OnInit {
   logInWithGoogle(platform: string) {
     this.resultMessage = '';
     platform = GoogleLoginProvider.PROVIDER_ID;
-    this.authService.signIn(platform).then(
+    this.authService.signIn(platform, { 'prompt': 'select_account' }).then(
       async (response) => {
-        if (response.email.split('@')[1] !== "ruralsourcing.com") return this.resultMessage = "Make sure you use your RSI email.";
+        if (response.email.split('@')[1] !== "ruralsourcing.com") {
```

Comment:
Do you have any other validation on the email response to make sure the emails are valid? I assume the plugin you're using has some handling for that but we want to make sure we aren't just trusting user input.

Reply:
I think it is safe to assume that Google will be enough to validate the user's email. Since they have to actually login to a real google account, and we don't have an optional form for the user to put in their own input.